ui/src/domain/jsonSchema.ts

```diff
@@ -0,0 +1,116 @@
+export type Attribute =
+  | StringAttribute
+  | IntegerAttribute
+  | NumberAttribute
+  | BooleanAttribute
+  | NullAttribute
+  | ObjectAttribute
+  | ArrayAttribute
+  | MultiAttribute;
+
```

Comment:
This module was sorted semantically, as the union above ^:

 * First, the primitive values: string, integer, number, and boolean
 * Then, the non-value: null
 * Then, the complex types: object, and array
 * Finally, the composite multi attribute
 
I don't like this new alphabetical sorting that starts with the attribute union and sends the string to the very bottom, below JsonSchema.

We meet on sorting rules and agreed on sorting the types' members and objects' properties, enforcing this with the linter, and we explicitly stated that we were not sorting the code alphabetically. I'd revert this file completely.

Reply:
This is why it's important to add commented sections and to apply some ordering that everyone can see.

Reply:
Revised & improved as discussed.